tools/mtouch/Makefile

```diff
@@ -277,7 +277,8 @@ SIMLAUNCHER_FRAMEWORKS =  \
 						\
 	-weak_framework ARKit					\
 	-weak_framework FileProvider			\
-	-weak_framework FileProviderUI
+	-weak_framework FileProviderUI			\
+	-weak_framework BusinessChat
```

Comment:
your missing the code to load the framework for XM
look at `src/Foundation/NSObject.mac.cs`